update-dois/src/main/resources/config.properties

```diff
@@ -1,12 +1,14 @@
-userTR=testReactomeMySQLUsername
-userGK=centralMySQLUsername
-passwordTR=testReactomePassword
-passwordGK=centralUsername
-hostTR=testReactomeHost
-hostGK=centralHost
-databaseTR=release_current
-databaseGK=central
-authorIdTR=personId
-authorIdGK=personId
-port=3306
-testMode=true
+release.database.user=releaseDatabaseUsername
+release.database.password=releaseDatabasePassword
+release.database.host=releaseDatabaseHost
+release_current.name=release_current
+release.database.port=3306
+
+curator.database.user=centralDatabaseUsername
+curator.database.password=centralDatabasePassword
+curator.database.host=centralDatabaseHost
+curator.database.name=central
+curator.database.port=3306
+
+personId=8985221
```

Comment:
A comment that the `personId` should be changed to the db id of the Person instance for the one running the release would be good.

Reply:
This is probably something that Jenkins should prompt the user for.

Reply:
Actually, the personId attribute should be left blank, that one just slipped through. I'll add comments